### DISCUSSION TITLE
`straight-pull-all` from the terminal?

### DISCUSSION THREAD

**Discussion by anoduck** [raxod502/straight.el#1164] [Q&A] [answered]
If I wanted to have straight pull and update my packages from the terminal, would just a simple `emacs --eval='(straight-pull-all)'` work? Or is this too simple?

How could it be achieved to do so non-interactively to use in a bash script?

**Comment by raxod502** [comment, marked as answer]
https://github.com/doomemacs/doomemacs has a large amount of wrapping around `straight.el` that makes it usable from the command line. #412 is for having this built-in upstream.

**Reply by anoduck** [reply]
Thank you, Radon.
